core/keyserver/keyserver.go

```diff
@@ -41,7 +41,7 @@ import (
 
 // Server holds internal state for the key server.
 type Server struct {
-	admin     adminstorage.Storage
+	admin     domainstorage.Storage
```

Comment:
Does it still make sense for this field to be named `admin`?

Reply:
Nope.  I'll rename it.